src/commands/fun/rollDice.ts

```diff
@@ -0,0 +1,45 @@
+import { Message } from 'discord.js';
+import { CommandoClient, CommandoMessage } from 'discord.js-commando';
+import { BaseCommand } from '../../utils/commands';
+
+class RollDiceCommand extends BaseCommand {
+  constructor(client: CommandoClient) {
+    super(client, {
+      name: 'roll-dice',
+      aliases: ['rd', 'roll', 'rolldice', 'dice-roll', 'diceroll', 'dice'],
+      group: 'fun',
+      args: [
+        {
+          key: 'sides',
+          prompt: 'enter the number of sides of the dice',
+          type: 'integer'
+        }
+      ],
+      memberName: 'dice-coin',
+      description: 'Roll a dice!',
+      examples: [`${client.commandPrefix}dice-roll`]
+    });
+  }
+
+  getRandomInt(max: number): number {
+    const roll = Math.floor(Math.random() * max);
+    if (roll == 0) {
+      return 1;
+    }
+    return roll;
+  }
+
+  async onRun(message: CommandoMessage, args: {sides: number;}): Promise<Message> {
+    const {sides} = args;
+    if (sides <= 0) {
+      return message.reply(`I cannot compute ` + sides + ' sides!');
+    }
+    else if (sides > 1000000) {
```

Comment:
Can change else if to an if since the previous of statement exits the function

Reply:
understood